src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '−') {
+    return one.minus(two).toString();
+  }
```

Comment:
- [ ] As requested by the previous review,  according to the project requirement,  Your `operate.js` file should be the same as the provided file.  Kindly Make sure to copy the entire data from this: [`operate.js file`](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js)

Check 

   |        Yours          |            Original.        |
---------------------:|:--------------------
![bad1](https://user-images.githubusercontent.com/79058364/164086807-f7984b62-5a5d-4531-9239-a4cd45ad57c9.png) | ![cool1](https://user-images.githubusercontent.com/79058364/164086870-44299e3a-72ae-4560-a83b-220dbdc510fd.png)

Reply:
Well resolved 👍🏽